js/movie_browser.js

```diff
@@ -1,2 +1,74 @@
-// API Docs at: 
-// http://www.omdbapi.com
\ No newline at end of file
+// API Docs at:
+// http://www.omdbapi.com
+$( document ).ready(function() {
+  console.log( "ready!" );
+});
+
+//on form submit this function runs two other functions
+function myFunction1(){
```

Comment:
No reason, I just hide my div in the css and on submit one function will run which in turn that function will run two other functions, one function controls the css of the div's one does the Ajax.